index.js

```diff
@@ -153,21 +185,28 @@ function redisStore(args) {
    * @method get
    * @param {String} key - The cache key
    * @param {Object} [options] - The options (optional)
+   * @param {boolean|Object} options.gzip - gzip configuration
    * @param {Function} cb - A callback that returns a potential error and the response
    */
   self.get = function(key, options, cb) {
     if (typeof options === 'function') {
       cb = options;
+      options = {};
+    }
+    options.parse = true;
+
+    var gzip = (options.gzip || options.gzip === false) ? options.gzip : redisOptions.gzip;
```

Comment:
Would  more simple comparison work the same? like:
 `var gzip = options.gzip ? options.gzip : redisOptions.gzip;`